Automattic/wp-calypso
# Hosting Command Palette: Add loading notices to actions

- Related to https://github.com/Automattic/dotcom-forge/issues/4707

## Proposed Changes

* Display a loading notice when taking an action.

## Testing Instructions
1. Go to `/sites` or any other page that our command palette is available
2. Press `cmd+k` on Mac and `ctrl+k` on Windows
3. Search for `Reset SSH/SFTP password` and press enter
4. Observe a temporal loading notice 
5. Observe a second success notice after a few seconds
6. Repeat step 3 searching for `Copy SSH connection string`

## Screencast


https://github.com/Automattic/wp-calypso/assets/779993/a1407a40-324a-4da5-a5c5-5e986c2eac68





## Pre-merge Checklist
